python/mujincontrollerclient/binpickingcontrollerclient.py

```diff
@@ -143,18 +144,20 @@ def StartPickAndPlaceThread(self, goaltype=None, goals=None, targetnamepattern=N
             'containername': regionname,
             'deletetarget': deletetarget,
             'debuglevel': debuglevel,
+            'resetIgnoreObjectsFromUpdateWithPrefix': resetIgnoreObjectsFromUpdateWithPrefix,
         }
         if goals is not None:
             taskparameters['orderedgoals'] = goals
             taskparameters['goaltype'] = goaltype
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, robotspeed=robotspeed, toolname=toolname, timeout=timeout, usewebapi=usewebapi)
 
-    def StopPickPlaceThread(self, resetExecutionState=True, resetStatusPickPlace=False, finishCode=None, timeout=10, usewebapi=None, fireandforget=False, **kwargs):
+    def StopPickPlaceThread(self, resetExecutionState=True, resetStatusPickPlace=False, finishCode=None, resetIgnoreObjectsFromUpdateWithPrefix=True, timeout=10, usewebapi=None, fireandforget=False, **kwargs):
         """stops the pick and place thread started with StartPickAndPlaceThread
         :param resetExecutionState: if True, then reset the order state variables. By default True
         :param resetStatusPickPlace: if True, then reset the statusPickPlace field of hte planning slave. By default False.
         :param finishCode: optional finish code to end the cycle with (if it doesn't end with something else beforehand)
+        :param resetIgnoreObjectsFromUpdateWithPrefix: if True, resets ignoreObjectsFromUpdateWithPrefix used to ignore some objects updates.
```

Comment:
@kanunikov-denys where do you add `resetIgnoreObjectsFromUpdateWithPrefix` in the command args?

Reply:
@rdiankov oh, sorry. You are correct I missed setting it for StopPickPlaceThread command. now is fixed.